Tasks/ServiceFabricUpdateAppVersions/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,11 +6,45 @@
   "loc.input.label.applicationPackagePath": "Application Package",
   "loc.input.help.applicationPackagePath": "Path to the application package. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) and wildcards can be used in the path.",
   "loc.input.label.versionSuffix": "Version Suffix",
-  "loc.input.help.versionSuffix": "Value to append to the version values in the manifest files.",
+  "loc.input.help.versionSuffix": "Value to append to the version values in the manifest files. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) and PowerShell expressions can be used.\n\nExample PowerShell expression: .($([DateTimeOffset]::UtcNow.ToString('u').Replace(':', '.')))",
```

Comment:
How sure are you that the task will always be implemented in PowerShell? Service Fabric isn't a PowerShell technology, correct?

Typically we try to limit PowerShell specific input interpretation, to powershell-specific tasks. For example, the PowerShell tasks runs ad-hoc powershell scripts, the Azure PowerShell task sets up an Azure PowerShell context and then runs an ad-hoc powershell script.

Customers have ways of achieving the same end you're after here. They can add a PowerShell task to their build definition that leverages a [## command](https://github.com/Microsoft/vsts-tasks/blob/master/docs/authoring/commands.md#logging-commands) to set a variable, then consume that variable in the input textbox of this task.

Is it conceivable that this task could one day have an implementation that runs on Linux/OSX? My inclination is that it would be better to stick with inputs that would work well on all platforms in the future, rather than plan for a sub-standard experience on OSX/Linux.